test/integration/node/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
Nit: Why are we not specifying approvers?

Reply:
emm. Actually, I don't know who I should specify. sig-node or sig-scheduling. Can you give me hint?

Reply:
Let me fill it.

Reply:
Who I need to speficy? It seems like sig-node-maintainers is incorrect.

Reply:
`sig-node-approvers` is the proper alias.